wp-includes/IXR/class-IXR-server.php

```diff
@@ -98,7 +98,7 @@ function call($methodname, $args)
         $method = $this->callbacks[$methodname];
 
         // Perform the callback and send the response
-        if (count($args) == 1) {
+        if (null !== $args && count($args) == 1) {
```

Comment:
Just FYI: this will still give issues if `$args` is a scalar or an object which doesn't implement `Countable`.